anaconda.spec.in

```diff
@@ -112,13 +112,15 @@ Requires: subscription-manager >= %{subscriptionmanagerver}
 # which is apparently great for containers but unhelpful for the rest of us
 Requires: cracklib-dicts
 
+%if ! 0%{?rhel}
 Requires: teamd
```

Comment:
If the installer doesn't actually depend on these packages (it looks like it can run without them), could they be moved to `install-env-deps` instead of keeping them in `core`?

Reply:
We depend on this package. It will crash in case you will try to use this functionality but for the branching purpose we agreed to approach it this way to avoid unnecessary dependencies drag in for branching. Cleaning of the code needs to be done and will be done later.

Reply:
What a mess...